lib/swagger_yard/swagger.rb

```diff
@@ -167,10 +167,7 @@ def property(prop)
         unless h['$ref']
           h["description"] = prop.description if prop.description && !prop.description.strip.empty?
           if prop.nullable
-            h["x-nullable"] = true
-            if h["type"]
-              h["type"] = [h["type"], "null"]
```

Comment:
Should type still be set as a non array property in this case?

Reply:
nope. the non array property is set somewhere else (I don't know where😅). This thing will rewrite 

type: string
=> 
type: [string, null]

Reply:
Yep I see now, it was just updating the type in this casa